v2/tools/generator/internal/config/type_transformer.go

```diff
@@ -333,6 +334,25 @@ func (r PropertyTransformResult) String() string {
 	)
 }
 
+// Log creates a log message for the transformation
+func (r PropertyTransformResult) Log(log logr.Logger) {
+	if r.Removed {
+		log.V(2).Info(
```

Comment:
You answered the "What does (v1) mean" elsewhere, but I do still think it's weird that these logs are different verbosities. What led to that decision?

Reply:
Originally both messages were at `V(3)` which would be invisible with the new system, so I pulled them up to `V(2)`. 

I don't recall the exact reason, but I then changed the transformation to `V(1)` because I thought it was more important.

Reply:
Can change both to `V(2)` if you like. We have a LOT of removals, so changing both to `V(1)` would be noisy.